### PR TITLE
*: unify wru,rru to ru

### PR DESCRIPTION
**Pull request by nolouch**
Signed-off-by: nolouch <nolouch@gmail.com>

### What problem does this PR solve?

Issue Number: close https://github.com/pingcap/tidb/issues/40883

Problem Summary:

### What is changed and how it works?
unify wru/rru, use ru as unit.
### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [ ] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

Please refer to [Release Notes Language Style Guide](https://pingcap.github.io/tidb-dev-guide/contribute-to-tidb/release-notes-style-guide.html) to write a quality release note.

```release-note
None
```

### CODE REVIEW

File: parser/parser_test.go [hunk 1]
```diff
@@ -3647,17 +3647,15 @@ func TestDDL(t *testing.T) {
 		{"create resource group x cpu ='8c'", true, "CREATE RESOURCE GROUP `x` CPU = '8c'"},
 		{"create resource group x region ='us, 3'", false, ""},
 		{"create resource group x cpu='8c', io_read_bandwidth='2GB/s', io_write_bandwidth='200MB/s'", true, "CREATE RESOURCE GROUP `x` CPU = '8c' IO_READ_BANDWIDTH = '2GB/s' IO_WRITE_BANDWIDTH = '200MB/s'"},
-		{"create resource group x rru_per_sec=2000", true, "CREATE RESOURCE GROUP `x` RRU_PER_SEC = 2000"},
-		{"create resource group x wru_per_sec=200000", true, "CREATE RESOURCE GROUP `x` WRU_PER_SEC = 200000"},
-		{"create resource group x rru_per_sec=2000 wru_per_sec=200000", true, "CREATE RESOURCE GROUP `x` RRU_PER_SEC = 2000 WRU_PER_SEC = 200000"},
+		{"create resource group x ru_per_sec=2000", true, "CREATE RESOURCE GROUP `x` RU_PER_SEC = 2000"},
+		{"create resource group x ru_per_sec=200000", true, "CREATE RESOURCE GROUP `x` RU_PER_SEC = 200000"},
```
Comment (glorv) [inline parser/parser_test.go:3651]: nit: This two cases are almost the same now.

**Review by glorv — APPROVED** [review]
LGTM

File: parser/parser_test.go (hunk 1 above)
Comment (nolouch) [reply parser/parser_test.go:3651]: I will use one with `burst case` after this pr merged.

**Review by BornChanger — APPROVED** [review]

File: privilege/privileges/privileges_test.go [hunk 2]
```diff
@@ -2603,14 +2603,13 @@ func TestResourceGroupAdminDynamicPriv(t *testing.T) {
 		`GRANT USAGE ON *.* TO 'resource_group_user'@'%'`,
 		`GRANT RESOURCE_GROUP_ADMIN ON *.* TO 'resource_group_user'@'%'`))
 
-	tk2.MustExec("CREATE RESOURCE GROUP test RRU_PER_SEC = 666")
-	tk2.MustExec("CREATE RESOURCE GROUP test2 WRU_PER_SEC = 999")
```
Comment (glorv) [inline privilege/privileges/privileges_test.go:2607]: Why drop this line

Comment (nolouch) [reply privilege/privileges/privileges_test.go:2607]: fixed. miss understand `test2` in here.

**Comment by nolouch** [comment]
updated client，ref https://github.com/pingcap/tidb/pull/40884

**Pull request merged**